content/docs/introducing-jsx.md

```diff
@@ -68,88 +68,88 @@ ReactDOM.render(
+Մի դրեք ապաթարցեր ձևավոր փակագծերի շուրջ երբ ներդրված է JavaScript արտահայտություն ատրիբուտում։ Դուք պետք է կամ օգտագործեք ապատարցեր (տողային արժեքների համար) կամ ձևավոր փակագծեր (արտահայտությունների համար), բայց ոչ երկուսը միասին նույն ատրիբուտում։
 
->**Warning:**
+>**Զգուշացում.**
 >
->Since JSX is closer to JavaScript than to HTML, React DOM uses `camelCase` property naming convention instead of HTML attribute names.
+>Քանի որ JSX-ը ավելի մոտ է JavaScript-ին քան HTML-ին, ReactDOM-ը HTML ատրիբուտների անուների փոխարեն օգտագործում է `ուղտաԳիր` հատկությունների անունների պայմանավորվածություն։
 >
->For example, `class` becomes [`className`](https://developer.mozilla.org/en-US/docs/Web/API/Element/className) in JSX, and `tabindex` becomes [`tabIndex`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/tabIndex).
+>Օրինակ. `class`-ը JSX-ում դառնում է [`className`](https://developer.mozilla.org/en-US/docs/Web/API/Element/className)<sub>`eng`</sub>, `tabindex`-ը դառնում է [`tabIndex`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/tabIndex)<sub>`eng`</sub>։
 
-### Specifying Children with JSX {#specifying-children-with-jsx}
+### Զավակների հայտարարումը JSX-ում {#specifying-children-with-jsx}
 
-If a tag is empty, you may close it immediately with `/>`, like XML:
+Եթե թեգը դատարկ է, դուք կարող եք անմիջապես փակել այն սրանով `/>`, ինչպես XML-ում.
 
 ```js
 const element = <img src={user.avatarUrl} />;
 ```
 
-JSX tags may contain children:
+JSX թեգերը կարող են պարունակել զավակներ.
 
 ```js
 const element = (
   <div>
-    <h1>Hello!</h1>
-    <h2>Good to see you here.</h2>
+    <h1>Ողջույն!</h1>
+    <h2>Ուրախ ենք Ձեզ տեսնել այստեղ։</h2>
   </div>
 );
 ```
 
-### JSX Prevents Injection Attacks {#jsx-prevents-injection-attacks}
+### JSX-ը կանխում է ներարկման հարձակումները {#jsx-prevents-injection-attacks}
 
-It is safe to embed user input in JSX:
+Ապահով է ներդնել օգտագործողի input-ը JSX-ում.
 
 ```js
 const title = response.potentiallyMaliciousInput;
 // This is safe:
 const element = <h1>{title}</h1>;
 ```
 
-By default, React DOM [escapes](https://stackoverflow.com/questions/7381974/which-characters-need-to-be-escaped-on-html) any values embedded in JSX before rendering them. Thus it ensures that you can never inject anything that's not explicitly written in your application. Everything is converted to a string before being rendered. This helps prevent [XSS (cross-site-scripting)](https://en.wikipedia.org/wiki/Cross-site_scripting) attacks.
+Լռությամբ, ReactDOM-ը էկրանավորում է JSX-ում ներդրված ցանկացած արժեք մինչ նրանց արտապատկերումը։ Այդպիսով նա երաշխավորում է, որ դուք երբեք չեք կարող «ներարկել» այն ինչ ակնհայտորեն գրված չի ձեր հավելվածում։ Ամեն ինչ ձևափոխվում է տողի մինչ արտապատկերվելը։ Սա թույլ է տալիս կանխել [XSS (cross-site-scripting)](https://en.wikipedia.org/wiki/Cross-site_scripting)<sub>`eng`</sub> հարձակումները։
```

Comment:
https://stackoverflow.com/questions/7381974/which-characters-need-to-be-escaped-on-html ակտիվ լինկ ա, բայց չկա